client/settings/survey-modal/style.scss

```diff
@@ -0,0 +1,35 @@
+.survey-section {
+	.components-modal__header {
+		margin-bottom: 16px;
+	}
+	.components-modal__content {
+		padding-bottom: 16px;
+	}
```

Comment:
This is not being honored. (also continuation from comment above)
![Screen Shot 2021-07-22 at 3 22 34 PM](https://user-images.githubusercontent.com/273592/126704676-d15de6a1-512f-4026-aefc-42a30b86a48d.png)


Remove?

Reply:
Yeah, let's remove for consistency.